python/pyarrow/tests/test_pandas.py

```diff
@@ -25,12 +25,14 @@
 from datetime import date, datetime, time, timedelta, timezone
 
 import hypothesis as h
-import hypothesis.extra.pytz as tzst
+try:
+    import hypothesis.extra.pytz as tzst
```

Comment:
I _think_ we could also use the `timezones` strategy from `pyarrow.tests.strategies` (which is already import as `past`), so then we don't have to redefine it here with the `if tzst: .. else: ..`